app/src/main/java/com/chatgptlite/wanted/data/remote/ConversationRepository.kt

```diff
@@ -5,5 +5,5 @@ import com.chatgptlite.wanted.models.ConversationModel
 interface ConversationRepository {
     suspend fun fetchConversations() : MutableList<ConversationModel>
     fun newConversation(conversation: ConversationModel) : ConversationModel
-    fun deleteConversation()
+    fun deleteConversation(index: Int)
```

Comment:
I'd prefer to use conversationId, it will make sure which conversation is deleted than using index

Reply:
Yes, I got the feeling that would not be the best approach(The way I was thinking about this: If I add a new conversation and than try to delete after a sec this would require to recall the firebase to find the documentPath id to able to remove it). I have created this code before the message delete function where I'm using the  conversationId to remove message from firebase. I will correct this.